src/taco-remote-lib/ios/iosBuild.ts

```diff
@@ -73,6 +75,50 @@ class IOSBuilder extends Builder {
         return Q({});
     }
 
+    protected collectVersionInfo(): Q.Promise<any> {
+        var self: IOSBuilder = this;
+
+        // Get the version of cordova-ios and XCode currently installed
+        let execDeferred: Q.Deferred<string> = Q.defer<string>();
+        child_process.exec("xcodebuild -version & cordova platforms version ios", { cwd: this.currentBuild.appDir } , function (error, stdout, stderr) {
```

Comment:
This will run `xcodebuild -version` and `cordova platforms version ios` concurrently. There might possibly be a race condition if they get their output interleaved? Might be better to use `&&` instead to sequence them, or to do it in 2 separate `exec` instances